src/View/NotAlone.elm

```diff
@@ -17,11 +18,11 @@ view model =
     div []
         [ containerContent
             [ header []
-                [ h1 [ css [ pageHeadingStyle ] ] [ text (t NotAloneTitle) ]
+                [ h1 [ css [ pageHeadingStyle ], id "focus-target", tabindex -1 ] [ text (t NotAloneTitle) ]
                 ]
             ]
         , container
-            [ ul [ css [ gridStyle ] ]
+            [ ul [ css [ gridStyle ], ariaLive "polite" ]
```

Comment:
? what's this `polite` ?

Reply:
The aria-live makes the content be read out when the page changes. It can take different urgencies and "polite" is the normal one, as opposed to assertive, or off.